### DISCUSSION TITLE
Can't run the project from .exe to give the args

### DISCUSSION THREAD

**Discussion by m-hilani** [MuhammadSulaiman001/opengl-lab#24] [Q&A] [answered]
Hi Mister,
I can't use msbuild althogh i added it to Path 
![Screenshot 2023-12-07 213256](https://github.com/MuhammadSulaiman001/opengl-lab/assets/148714322/41faa863-378a-476e-ac10-b2f89d82e8b0)

![Screenshot 2023-12-07 214403](https://github.com/MuhammadSulaiman001/opengl-lab/assets/148714322/0f94e7eb-1c87-45cb-9b3b-5733d45e8ecf)

2. after debugging it using visual studio i faced abort() error after opening .exe file to pass the numbers
![Screenshot 2023-12-07 214445](https://github.com/MuhammadSulaiman001/opengl-lab/assets/148714322/7a6f6c08-93ea-4a55-a056-13f2f58a8062)

thanks

**Comment by MuhammadSulaiman001** [comment, marked as answer]
Hi @m-hilani..
1. As mentioned [here](https://github.com/MuhammadSulaiman001/opengl-lab/blob/master/docs/Getting%20Started.md#quick-start), You should add the path to PATH env variable... you are adding the path as **new** variable (name-value), this is wrong.
Find the name PATH in the list of system variables, click **Edit**, then click New and add the path.
2. You have to use `int main(int argc, chear* argv[])` declaration.. please try passing the args in a new empty project, it should work (the exception might be thrown due to some other initialization code before the execution reach the main function).
